src/bo4e/enum/eeg_vermarktungsform.py

```diff
@@ -0,0 +1,18 @@
+# pylint:disable=missing-module-docstring
+
+from bo4e.enum.strenum import StrEnum
+
+
+class EEGVermarktungsform(StrEnum):
+    """
+    Diese Enum repräsentiert die Vermarktungsformen nach dem EEG.
+    """
+
+    AUSFALLVERGUETUNG = "AUSFALLVERGUETUNG"  #: Ausfallvergütung
+    # Ausfallvergütung für den Fall, dass andere Vermarktungsmethoden nicht verfügbar sind
```

Comment:
Die Kommentare sind hier doppelt, also ich meine jeweils das obere: #: Ausfallvergütung braucht man nicht auch noch, wenn man die Langfassung hat, oder ich habe irgendeine Dokuemtationsänderung nicht mitbekommen. :-)

Reply:
e053ffcf5190110d440809e9c235e46d71061088

Reply:
Bitte sicherstellen, ob der Kommentar in der Doku auftaucht. Bin mir relativ sicher, dass das Autodoc Feature von Sphinx nur `#:` über der Zeile, `#` in der Zeile und `""" ... """` unter der Zeile kann.

Reply:
Ah sehr gut zu wissen, ja stimmt so hatten wir es an anderen Stellen glaub ich auch gehandhabt, ich hab bisher einfach immer geschaut wie es an anderen Stellen gemacht wird